### PR TITLE
feat: product-listing

### PR DESCRIPTION
**Pull request by KaustubhM1997**
image
image

### CODE REVIEW

File: product-listing/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+input {
+
+    font: inherit;
+}
+
+.filter-category .inp-checkbox {
+
+    display: flex;
+    /* justify-content: flex-start; 
+    align-items: center; */
```
Comment (gopalbbharadva) [inline product-listing/product-listing.css]: Suggetion: Remove unused comments

File: product-listing/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    object-fit: contain;
+}
+
+
+.card-text {
+
+    font-size: 0.9rem;
+    font-weight: 600;
+    margin-top: 0.9rem;
+    margin-left: 0.5rem;
```
Comment (gopalbbharadva) [inline product-listing/product-listing.css]: Improvement: It can be done like margin: 0.9rem 0 0 0.5rem

**Review by gopalbbharadva — APPROVED** [review]

**Comment by suryxks** [comment]
+1 to what gopal has mentioned 
other than that ui looks good

**Review by suryxks — APPROVED** [review]

**Comment by preethasri** [comment]
UI looks good

File: product-listing/product-listing.css (hunk 2 above)
Comment (KaustubhM1997) [reply product-listing/product-listing.css]: Thanks, Gopal. I've now used the shorthand property.

**Pull request merged**